spec/unit/type/file_spec.rb

```diff
@@ -1103,7 +1103,7 @@
     end
 
     it "should cache the stat instance" do
-      expect(file.stat).to equal(file.stat)
+      expect(file.stat.object_id).to equal(file.stat.object_id)
```

Comment:
I think rspec `equal` already tests that they refer to the same object: https://relishapp.com/rspec/rspec-expectations/v/3-7/docs/built-in-matchers/equality-matchers

That said I think it's more readable to make `object_id` explicit, though in that case you probably want `expect(..).to eql(file.stat.object_id)`